test/jdk/com/sun/tools/attach/warnings/DynamicLoadWarningTest.java

```diff
@@ -123,8 +123,6 @@ void testLoadOneJvmtiAgent(OnAttachAction loadJvmtiAgent1) throws Exception {
                 .whenRunning(loadJvmtiAgent1)
                 .stderrShouldNotContain(JVMTI_AGENT_WARNING);
 
-        // test behavior on platforms that can detect if an agent library was previously loaded
-        if (!Platform.isAix()) {
```

Comment:
You need to fix the indentation of the old block.

Reply:
I followed your suggestion here.